Header header
int32 min_angle
int32 max_angle
